dpnp/backend/extensions/blas/dot_common.hpp

```diff
@@ -23,30 +23,26 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 //*****************************************************************************
 
-#include <pybind11/pybind11.h>
+#pragma once
+
+#include <oneapi/mkl.hpp>
 
 // dpctl tensor headers
 #include "utils/memory_overlap.hpp"
 #include "utils/output_validation.hpp"
+#include "utils/type_dispatch.hpp"
 #include "utils/type_utils.hpp"
 
-#include "dot.hpp"
 #include "types_matrix.hpp"
 
-#include "dpnp_utils.hpp"
-
 namespace dpnp
 {
 namespace backend
 {
 namespace ext
 {
-namespace blas
+namespace dot
```

Comment:
It would be better to add inner namespace `dpnp::backend::ext::blas::dot` rather than renaming old one.